### DISCUSSION TITLE
how to manage inbalance data from spacy?

### DISCUSSION THREAD

**Discussion by SergioMarreroMarrero** [explosion/spaCy#7831] [Help: Coding & Implementations] [answered]
Hi.

I have an imbalanced dataset. Before jumping to make my custom undersampling/oversampling using other libraries I wanted to ask if there is a way for managing it from spacy.

Thank you.

**Comment by polm** [comment, marked as answer]
spaCy doesn't have any special functions for imbalanced data as such, though there is functionality for [data augmentation](https://spacy.io/usage/training#data-augmentation), which can help with things like orthographic variation.
